src/policy/fees.cpp

```diff
@@ -957,12 +956,16 @@ void CBlockPolicyEstimator::FlushFeeEstimates()
     }
 }
 
+// The current format written, and the version required to read. Must be
+// increased to at least 279900+1 on the next breaking change.
+constexpr int CURRENT_FILE_VERSION{149900};
+
 bool CBlockPolicyEstimator::Write(AutoFile& fileout) const
 {
     try {
         LOCK(m_cs_fee_estimator);
-        fileout << 149900; // version required to read: 0.14.99 or later
-        fileout << CLIENT_VERSION; // version that wrote the file
+        fileout << CURRENT_FILE_VERSION;
+        fileout << 279900; // version that wrote the file. Used to be CLIENT_VERSION. Currently unused dummy field.
```

Comment:
Storing `nVersionThatWrote` seems somewhat useful for manually debugging a corrupt fee data file, and figuring out how the bug occurred. If we're just storing a constant, then the value isn't going to be useful, so it seems better to just write `(int)0` instead -- that's the amount of information it will convey... In any event, "version that wrote the file" is not accurate anymore if we're making it something different from `CLIENT_VERSION`.